client/src/components/BlockLinkFieldActions/BlockLinkFieldActions.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+import { inject } from 'lib/Injector';
+import classnames from 'classnames';
+
+class BlockLinkFieldActions extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleChangeValue = this.handleChangeValue.bind(this);
+  }
+
+  /**
+   * @param {String} value
+   * @returns {Object} One of props.actions.
+   */
+  getOptionByValue(value) {
+    return this.props.actions.find(action => action.value === value);
+  }
+
+  /**
+   * @param {Event} event
+   * @returns {Promise|null}
+   */
+  handleChangeValue(event) {
+    let promise = null;
+
+    // Make sure a valid option has been selected.
+    const option = this.getOptionByValue(event.target.value);
+    if (option === null) {
```

Comment:
I don't think this will ever be `true`, because `Array.find()` returns `undefined` when not found.
I recommend the typeof check for undefined.